uk.co.compucorp.civicrm.appraisals/tests/phpunit/api/v3/Appraisals/AppraisalsApiTest.php

```diff
@@ -1,13 +1,16 @@
 <?php
 
-require_once 'CiviTest/CiviUnitTestCase.php';
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
 
 /**
- * Test for Appraisal API calls.
+ * Class CRM_HRVisa_ActivityTest
+ *
+ * @group headless
  */
-class AppraisalsApiTest extends CiviUnitTestCase {
+class api_v3_Appraisals_AppraisalsApiTest extends CiviUnitTestCase implements HeadlessInterface , TransactionalInterface {
     const DAY = 86400;
```

Comment:
I know this has nothing to do with your PR, but here is a nice example of how a bad name is as bad as an magic number somewhere in code.